src/sentry/monitors/models.py

```diff
@@ -312,14 +312,8 @@ def ensure_environment(
         # TODO: assume these objects exist once backfill is completed
         environment = Environment.get_or_create(project=project, name=environment_name)
 
-        monitorenvironment_defaults = {
-            "status": monitor.status,
-            "next_checkin": monitor.next_checkin,
-            "last_checkin": monitor.last_checkin,
-        }
-
         return MonitorEnvironment.objects.get_or_create(
-            monitor=monitor, environment=environment, defaults=monitorenvironment_defaults
```

Comment:
I think we should add some helper code to set intelligent defaults on `next_checkin` keyed off the `monitor`. also, i'm not sure we really want to be setting individual `MonitorEnvironment` as `DISABLED` quite yet (even though there are 230 in production) so we can have flexibility with the feature moving forward. I don't think there's a way to activate individual `MonitorEnvironments` currently (outside of a successful/failed check-in)

Reply:
What kind of logic are you thinking?

I think we just want it to be unset on creation, it will be updated with values from the checkin that happens right after it's created no